ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -723,9 +793,9 @@ class NGraphEncapsulateOp : public OpKernel {
 #endif
       current_ng_tensor = GetCurrentNgTensor(
           current_dst_ptr, last_dst_ptr, last_ng_tensor, true, ng_exec,
-          op_backend, ng_element_type, ng_shape);
-
-      current_ng_tensor->set_stale(true);
```

Comment:
removing this, because `current_ng_tensor`'s staleness is set in `GetCurrentNgTensor`. So this line is not needed